x-pack/docs/en/ml/troubleshooting.asciidoc

```diff
@@ -114,3 +117,38 @@ start the failing nodes you must downgrade the nodes then fix the problem per
 above.
 
 See also {stack-ref}/upgrading-elastic-stack.html[Upgrading the Elastic Stack].
+
+[[ml-stopjobs]]
+
+=== Problems deleting {ml} jobs
+
+Before you can delete a job, you must delete its {dfeeds} and close the job. If 
+you still encounter problems deleting the job, you might need to take extra 
+steps.
+
+*Symptoms:*
+
+* The {dfeed} was stopped and deleted successfully. When you try to delete the 
+job, however, it fails. For example, you receive an error message that says the 
+job is in a failed state. 
+
+*Resolution:*
+
+Try to delete the job again and specify the `force` parameter.  For example:
+
+[source,js]
+--------------------------------------------------
+DELETE _xpack/ml/anomaly_detectors/total-requests?force=true
+-------------------------------------------------- 
+// CONSOLE
+// TEST[setup:server_metrics_job]
+
+If the job persists, check whether there are any hung {ml} processes. For 
+example use the `ps` command and grep for the autodetect process, which is the 
+C++ process that runs the {ml} code. Kill these autodetect processes then delete 
+the job. 
+
+If the job still persists, you can delete the results index. If other jobs 
```

Comment:
@droberts195 Should we ever document this? If we do, then I'd word it much more defensively..

Only delete the results index if you know that the job in question is the only one using this index. Otherwise all results for all jobs will be lost.

Reply:
The whole section is listing workarounds for problems that should never happen, so if we have the section at all I think it’s reasonable to include this too. But I agree it needs to be worded to make the dangers clear.